### PR TITLE
updating imports

### PR DESCRIPTION
**Pull request by ericcater**
Should be no @odst/types or other GQL imports in services and no prisma imports in resolver

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -30,6 +30,28 @@
       "files": ["*.js", "*.jsx"],
       "extends": ["plugin:@nrwl/nx/javascript"],
       "rules": {}
+    },
+    {
+      "files": ["*.resolver.ts"],
```
Comment (yurisim) [inline .eslintrc.json:35]: Add a message to the error to explain why it shouldn't be there, see,

image

https://eslint.org/docs/rules/no-restricted-imports

**Review by yurisim — CHANGES_REQUESTED** [review]
add a message to the rules, otherwise good job

**Review by yurisim — APPROVED** [review]

File: .eslintrc.json [hunk 2]
```diff
@@ -30,6 +30,39 @@
...
+      //Bug: does not run against frontend files, which isn't desired but it is expected
+      "files": ["*.service.ts"],
+      "rules": {
+        "no-restricted-imports": [
+          "error",
+          {
+            "patterns": [
+              {
+                "group": ["@odst/types/*"],
+                "message": "Do not import types from @odst/types in servicer, use prisma types instead (or greneted gql if in frontend)"
```
Comment (yurisim) [inline .eslintrc.json:60]: :( greneted

**Review by yurisim — CHANGES_REQUESTED** [review]
greneted

File: .eslintrc.json (hunk 2 above)
Comment (ericcater) [reply .eslintrc.json:60]: I know what I siad

**Pull request merged**